daktari/checks/test_ssh.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from daktari.checks.ssh import is_ssh_configured_to_use_macos_keychain
+
+
+class TestSSH(unittest.TestCase):
+    def test_ssh_configured_correctly(self):
+        self.assertTrue(is_ssh_configured_to_use_macos_keychain("./daktari/checks/good_ssh_config"))
+
+    def test_ssh_not_configured(self):
```

Comment:
Sorry to be a pain, but it might be good to add an additional test case for where `UseKeychain yes` is present, but `IgnoreUnknown UseKeychain` is not 🙏